docs/instrument-traces/dotnet-app.md

```diff
@@ -0,0 +1,129 @@
+# .Net App
+
+### Auto-instrument sample Dotnet App for traces
+
+In this tutorial we will go through the steps to auto instrument a .Net app to send traces to Siglens.
+
+### Prerequisites
+- Siglens instance should be running on localhost with ingest port-4318. To do so you need to change the ingest port of Siglens to `4318` in `server.yaml`
+- .Net app (refer the documentation below if you don't have the setup for .Net app)
+
+### Set up for .Net application
+
+Note: You need to have .NET SDK installed
```

Comment:
Since having the .NET SDK is a prerequisite, put it in the Prerequisites section. You should probably also link to some external website that has instructions on how to install the .NET SDK